### PR TITLE
Game Functionalities

### PR DESCRIPTION
**Pull request by antoniomf97**
Regards the stickian gameplay description, focused on the functionality (buildings, units, technology tree).

### CODE REVIEW

File: documents/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Stickian Empire, The Game
+
+The Stickian Empire is a Massively Multiplayer Online (MMO) Real Time Strategy (RTS) Browser Game. Players can build their empire, join alliances and fight against others until achieve victory.
```
Comment (luisferreira32) [inline documents/README.md]:
```suggestion
The Stickian Empire is a Massively Multiplayer Online (MMO) Real Time Strategy (RTS) Browser Game. Players can build their empire, join alliances and fight against others until victory is achieved. 
```

File: documents/TECHTREE.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Technology Tree
+
+The technological tree is subdivided into five main categories, each associated with a color:
+1. <ins>**Resources and Wealth**</ins> 
+[<span style="color:#009900">green | #009900</span>]: regards all topics on resources production, storage, trade and management;
+
+2. <ins>**Science and Technology**</ins> [<span style="color:#1155cc">blue | #1155cc</span>]: regards to all topics on science and its production, as well as new technology to unlock different assets
```
Comment (luisferreira32) [inline documents/TECHTREE.md]: Not sure this is doing your intended purpose, check the "rich diff" on the PR

File: documents/BUILDINGS.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Buildings
+
+[Reference Document](https://docs.google.com/spreadsheets/d/1H_UoOrvQGwsS0GIzRcd_AlPuNHbIsHRAz4Llo7vYrSU/edit?usp=sharing)
```
Comment (luisferreira32) [inline documents/BUILDINGS.md]: No, we should not be sharing a random public google sheets. Either it's here or it's private.

File: documents/UNITS.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# Stickian Units
+
+[Reference Document](https://docs.google.com/spreadsheets/d/1ufbMvYJxB7khexCzSmpdUhZsics2AKDPn-ZO1j_ako4/edit?usp=sharing)
```
Comment (luisferreira32) [inline documents/UNITS.md]: Same comments applies here

**Review by luisferreira32 — APPROVED** [review]
Address the comments first then merge squash

File: documents/README.md (hunk 1 above)
Comment (antoniomf97) [reply documents/README.md]: That's okay

**Pull request merged**